cellprofiler/modules/medianfilter.py

```diff
@@ -36,8 +36,8 @@ def create_settings(self):
             value=3,
             minval=0,
             doc="""
-                Patch size for computing the median filter. Must be odd. Use a window with a small size to
-                remove small elements of noise. A larger window will remove larger elements of noise at the
+                Dimension in each direction for computing the median filter. Must be odd. Use a window with a small size to
```

Comment:
Typically we have multi-line docstrings aligned all the way to the left (because I think the regular docstrings keep the tabs if you don't do this)